clients/venice-pulsar/src/main/java/com/linkedin/venice/pulsar/sink/VeniceSink.java

```diff
@@ -231,6 +232,8 @@ private static Map<String, String> getConfig(VeniceSinkConfig veniceCfg, String
     config.put(configPrefix + VENICE_PUSH_TYPE, Version.PushType.INCREMENTAL.toString());
     config.put(configPrefix + VENICE_STORE, veniceCfg.getStoreName());
     config.put(configPrefix + VENICE_AGGREGATE, "false");
+    config.put(VALIDATE_VENICE_INTERNAL_SCHEMA_VERSION, "false");
```

Comment:
@dlg99 in some test environment I had an error and I had to disable this feature.

Maybe we could follow up and let Pulsar users pass a generic map of additional settings to tune the internal Writer.

Reply:
This change seems to break the integration tests, so I rolled it back and I added a generic map<string, string> to customize the config in case of problems.

I don't have anymore the logs of the error